src/test/java/de/tum/in/www1/artemis/exercise/programmingexercise/ProgrammingExerciseFactory.java

```diff
@@ -237,21 +316,32 @@ public static BambooBuildResultNotificationDTO generateBambooBuildResult(String
     }
 
     /**
-     * Generate a Bamboo notification with build logs of various sizes
+     * Generate a Bamboo result notification DTO with build logs of various sizes. The successful boolean value is set to true if there are no failed test names.
```

Comment:
In the called method, you specify that it's ten build logs. Here, you don't. Is there a reason for it?

Reply:
It should also say 10 logs here - I added it now